layouts/partials/tw/search-articles.html

```diff
@@ -10,10 +10,11 @@
       <label class="sr-only" for="{{ $idInput }}">Search</label>
       <input
         id="{{ $idInput }}"
-        class="spl-input rounded-r-none"
+        class="spl-input scroll-mt-4 rounded-r-none"
```

Comment:
You can replace these classes with `.btn-signup-slim` to simplify brand/style maintenance